python/xorbits/_mars/dataframe/groupby/tests/test_groupby_execution.py

```diff
@@ -1878,3 +1878,140 @@ def test_series_groupby_rolling_agg(setup, window, min_periods, center, closed,
     mresult = mresult.execute().fetch()
 
     pd.testing.assert_series_equal(presult, mresult.sort_index())
+
+
+def test_groupby_nth(setup):
```

Comment:
`.nth(slice(Noen, -1))` has been added, but I think there's no easy way to make `nth` in `xorbits`  subscriptable like `pandas`,  since `nth` in pandas is registered as a `property` which carries the groupby information https://github.com/pandas-dev/pandas/blob/v2.1.1/pandas/core/groupby/groupby.py#L4044-L4136 , in xorbits, we just use `setattr` to load `nth` on dataframe, until we *call* `nth`, `nth` doesn't know anything about groupby information.